packages/toolkit/config/postcss.config.js

```diff
@@ -1,10 +1,24 @@
 const path = require('path');
+const glob = require('fast-glob');
+
+const { getTenUpScriptsConfig } = require('../utils');
 
 module.exports = ({ file, env }) => {
+	const projectConfig = getTenUpScriptsConfig();
+	const { globalStylesDir, globalMixinsDir } = projectConfig.paths;
+
+	const globalCssFiles = glob.sync(`${globalStylesDir}**/*.css`);
```

Comment:
Will this fail or just return an empty result if the global folder doesn't exist, or if it doesn't have any css files?

Reply:
In my testing it always returned an empty array